libflux/src/flux/semantic/parser/mod.rs

```diff
@@ -502,8 +507,8 @@ impl Parser<'_> {
         self.next();
         let mut token = self.next();
 
-        let mut req_args = HashMap::new();
-        let mut opt_args = HashMap::new();
+        let mut req_args = BTreeMap::new();
+        let mut opt_args = BTreeMap::new();
         let mut pipe_arg = None;
         let mut need_comma = false;
         loop {
```

Comment:
It might be good for us to get in the habit of adding explicit type aliases for the generic data structures we use. It can greatly aid in understanding when you can see specifically what the map is for.

Reply:
Realizing now after rebasing that this is the same code I changed in the semantic-btree-hashmap patch so they are aligned, which is good.